tests/e2e/E2eTests.csproj

```diff
@@ -20,11 +20,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autodesk.Forge.Core.E2eTestHelpers" Version="1.*" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Autodesk.Forge.Core.E2eTestHelpers" Version="1.0.3" />
```

Comment:
Just noticed it was `1.*` , I guess it is unnecessary to specify `1.0.3` then?

Reply:
Right. WE want to "float" this dependency. Always pick up the latest.